lib/src/meta.dart

```diff
@@ -0,0 +1,112 @@
+import 'geojson.dart';
+
+typedef GeomEachCallback = dynamic Function(
+  Geometry currentGeometry,
+  int featureIndex,
+  Map<String, dynamic> featureProperties,
+  BBox featureBBox,
+  dynamic featureId,
+);
+
+/// Iterate over each geometry in any GeoJSON object, similar to Array.forEach()
+void geomEach(dynamic geoJSON, GeomEachCallback callback) {
```

Comment:
When comparing the dartdoc with the [JS equivalent](https://github.com/Turfjs/turf/blob/master/packages/turf-meta/index.js#L486-L506) some documentation is lost. While dartdoc doesn't support documentation annotations, it's still a best practice to document the parameters (see https://dart.dev/guides/language/effective-dart/documentation#do-use-prose-to-explain-parameters-return-values-and-exceptions). That doc also showcases how to use example code with using backticks.

Reply:
I pushed a separate commit, do you think the updated docs work?